tests/test_chebyshevBoundedField.py

```diff
@@ -90,6 +90,8 @@ def setUp(self):
 
         array = np.arange(self.bbox.getArea(), dtype=np.float32).reshape(self.bbox.getDimensions())
         self.image = lsst.afw.image.ImageF(array)
+        self.fields = [lsst.afw.math.ChebyshevBoundedField(self.bbox, coeffs) for _, coeffs in self.cases]
+        self.product = lsst.afw.math.ProductBoundedField(self.fields)
```

Comment:
Should there be an additional test product where these are done in the opposite order, just to confirm commutitivity, or does that go without saying?

Reply:
I'm not really worried about commutivity except at the level of floating point math edge cases, and I don't think it's worth trying to deal with those.